src/components/UI/Button.js

```diff
@@ -1,14 +1,15 @@
 import PropTypes from 'prop-types';
 import React from 'react';
 import BsButton from 'react-bootstrap/lib/Button';
+import styled from 'styled-components';
 
 import LoadingIndicator from './LoadingIndicator';
 
 // Button
 //
 // <Button
 //   type='button|submit|reset'
-//   bsStyle='primary|success|danger|info|warning|link'
+//   bsStyle='primary|secondary|danger|outline|link'
 //   loading=true|false
 //   disabled=true|false
 //   onClick=function>
```

Comment:
Can we have a props interface instead of documenting the props here?

Reply:
Yeah for sure. plan to do that in another PR